notebooks/03-query/10-functions-ip.ipynb

```diff
@@ -185,6 +185,81 @@
     "display.sql(sql)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "d23b9a99-cb5e-4b38-b4f9-07aec117ba5e",
+   "metadata": {},
+   "source": [
+    "### New in Druid 29.0.0 - IPV6_MATCH function\n",
+    "IPV6 utilization is on the rise, Druid 29.0.0 introduces this function that allows you to match IPV6 addresses to a subnet.\n",
+    "\n",
+    "The following example loads a few IPV6 addresses and demonstrates this new matching functionality."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5b393b4d-0784-4250-bdad-09508b13fb5f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "REPLACE INTO \"example-ipv6\" OVERWRITE ALL\n",
```

Comment:
This should go to the top section where data is ingested.

Reply:
Not sure I'm following your logic... the table being loaded here is just relevant for the two examples of filtering so it seems more logical to have these three cells together. The difference is probably also related to the new Version strategy. Let's discuss.